src/Mantasflowers.WebApi/Validation/Order/PostOrderItemRequestValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using Mantasflowers.Contracts.Order.Request;
+
+namespace Mantasflowers.WebApi.Validation.Order
+{
+    public class PostOrderItemRequestValidator : AbstractValidator<PostOrderItemRequest>
+    {
+        public PostOrderItemRequestValidator()
+        {
+            RuleFor(x => x.Id)
+                .NotEmpty();
+
+            RuleFor(x => x.Quantity)
+                .NotEmpty()
+                .GreaterThan(0);
+
+            RuleFor(x => x.UnitPrice)
```

Comment:
What is this again? Price at which the item is bought?

Reply:
Yes. Probably could also be pulled out from DB, but it wasn't specified how we'd handle this.

Reply:
I think we had in mind the fact that the price could change and we want to have the original price for history. Makes sense in that case.